src/resources/css/tribe-events-full-mobile.css

```diff
@@ -175,6 +175,75 @@ a.tribe-events-gmap {
 	margin: 36px 0 0;
 }
 
+/* = Short List View Template Styles
+   @ Template: shortlist.php
+=============================================*/	
+.tribe-events-shortlist .tribe-events-list-separator-month {
+	margin-top: 20px;
+}
+.tribe-events-shortlist .tribe-events-event-image {
+	float: left;
+	margin: 0 3% 0 0;
+}
+.tribe-events-shortlist .tribe-events-event-image img {
+	width: auto;
+	max-height: 400px;
```

Comment:
I know the CSS files in the-events-calendar aren't a great reference for our future intentions, but going forward we're going to be alphabetizing CSS properties. We've started to do this in our other plugins and any new additions in here we'd love to see the properties alphabetized, too :grinning:

If you could make that change, that'd be great!